user/sonarqube.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Using SonarQube.com with Travis CI
+layout: en
+permalink: /user/sonarqube/
+---
+**[SonarQube.com](https://sonarqube.com)** is the cloud service based on [SonarQube](http://www.sonarqube.org), the widely adopted "continuous inspection" open source platform that allows developers to find bugs, vulnerabilities and code smells in more than 20 different languages. It offers all the dashboards and tools to continuously track modifications of your code to make sure your code will only get better and better over time.
+
+SonarQube.com is free for open-source projects. The only requirement is to have a GitHub account to connect to the service. It is then up to you to trigger the analysis, and this is where this Travis CI add-on simplifies your user experience.
+
+## Activate the add-on
+
+Simply add the following lines to your `.travis.yml` file:
+
+    addons:
+      sonarqube: true
+
+This will set up everything required to simply trigger the SonarQube analysis later on.
+
+## Trigger the analysis
+
+Triggering the analysis is done thanks to a **SonarQube Scanner** - just like everyone would do for any SonarQube analysis. Depending on the build technology that you are using, you will use [one SonarQube Scanner or another](http://redirect.sonarsource.com/doc/analyzing-source-code.html).  
+
+For example, using the standard [SonarQube Scanner](http://redirect.sonarsource.com/doc/install-configure-scanner.html), you can simply trigger the analysis this way:
+
+    script:
+      - sonar-scanner -Dsonar.login=$SONAR_TOKEN
```

Comment:
@BanzaiMan In this first version, yes. As mentioned in the last part of this page, this is obviously something that we should simplify in a 2nd version of the add-on. I will be working on the specification of this shortly so I'll get back to you at that point of time.